packages/web/pages/stake/index.tsx

```diff
@@ -120,3 +125,36 @@ export const Staking: React.FC = observer(() => {
 });
 
 export default Staking;
+
+// Delete all this once staking is released
+export async function getServerSideProps() {
+  const ldClient = LDClient.init(
+    process.env.NEXT_PUBLIC_LAUNCH_DARKLY_SDK_KEY || ""
```

Comment:
@mattupham @jonator @JoseRFelix Everyone will need to update their env.local with the develoopment SDK key for this to work